### DISCUSSION TITLE
How to add a Chapter

### DISCUSSION THREAD

**Discussion by devhot** [iina/iina#5192] [Q&A]
Does anyone know how to add a Chapter when watching the video

**Comment by low-batt** [comment]
Adding chapters requires editing the video file. IINA is a player, not a video editor. [FFmpeg](https://ffmpeg.org/) can be used to add chapters to a video file.

**Reply by devhot** [reply]
but ,I see that there is "Chapters" options in IINA, which confuse me much
<img width="230" height="72" alt="image" src="https://github.com/user-attachments/assets/330ec256-70f0-469c-aca1-a17fdd7da9c4" />

**Reply by low-batt** [reply]
The `Chapters` sub-menu allows you to go to a particular chapter in the video being played. As the vast majority of video files do not contain chapters, normally the `Chapters`  sub-menu will be empty. To see this IINA feature working you must find and play a video whose author added chapters to it. For whatever reason most authors do not add chapters to their video files.
